pymongo/server.py

```diff
@@ -196,11 +199,13 @@ def run_operation(
             elif operation.name == "explain":
                 res = docs[0] if docs else {}
             else:
-                res = {"cursor": {"id": reply.cursor_id, "ns": operation.namespace()}, "ok": 1}
+                # assert isinstance(reply, _OpReply)
```

Comment:
Is this assert needed?

Reply:
no sorry, i was trying to test something and forgot to remove it. thanks for catching that!